packages/plugins/i18n/server/migrations/field/migrate-for-bookshelf.js

```diff
@@ -18,6 +18,7 @@ const batchInsertInTmpTable = async ({ updatesInfo }, { transacting: trx }) => {
 const updateFromTmpTable = async ({ model, attributesToMigrate }, { transacting: trx }) => {
   const { collectionName } = model;
   if (model.client === 'pg') {
+    // IMPORTANT TODO: use postgres schema
```

Comment:
migration-for-bookshelf ? we probably didn't delete this file with the v4 release

Reply:
Yes it was intentionnal not to do it as it was supposed to be automatically be updated as soon as there was an edit or sth I believe